app/src/components/trace/SpanAnnotationsEditor.tsx

```diff
@@ -170,13 +172,21 @@ function NewAnnotationCard(props: NewAnnotationCardProps) {
   );
 }
 
+/**
+ * Exclude the explanation button from being focused via the focus manager
+ */
+const excludeExplanationButton: FocusManagerOptions["accept"] = (node) => {
+  return !node.matches("button.annotation-input-explanation");
```

Comment:
does tab -1 not work here?

Reply:
No because the explain button is already not in the tab order. It's only focused because it is the first element in the form and the react area focus manager targets it